Assets/Packages/com.studio23.ss2.bettercursormanager/Runtime/Core/Controllers/CursorEventController.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections.Generic;
+using Studio23.SS2.BetterCursor.Interfaces;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.InputSystem;
+
+namespace Studio23.SS2.BetterCursor.Core
+{
+    public class CursorEventController : MonoBehaviour
+    {
+        private IHoverable _currentHoveringObject;
+        private IHoverable _lastHoveredObject;
```

Comment:
No getter for hovered object. Also, having two fields can be confusing. Code suggests they will always be equal. when accessed from outside. So one var should be enough.